gnovm/pkg/gnolang/store.go

```diff
@@ -865,6 +866,27 @@ func (ds *defaultStore) GetMemFile(path string, name string) *gnovm.MemFile {
 	return memFile
 }
 
+// FindPathsByPrefix retrieves all paths starting with the given prefix.
+func (ds *defaultStore) FindPathsByPrefix(prefix string) []string {
+	startKey := []byte(backendPackagePathKey(prefix))
+
+	// create endkey by incrementing last byte of startkey
+	endKey := make([]byte, len(startKey))
+	copy(endKey, startKey)
```

Comment:
```suggestion
	endKey := slices.Clone(startKey)
```

Reply:
every time you suggest me to use the `slices` package, it makes me feel like a boomer 😢